test/distributed/org/apache/cassandra/distributed/upgrade/IndexUnknownIgnoreTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.distributed.upgrade;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import static org.apache.cassandra.config.CassandraRelevantProperties.INDEX_UNKNOWN_IGNORE;
+import static org.apache.cassandra.distributed.api.Feature.GOSSIP;
+import static org.apache.cassandra.distributed.api.Feature.NETWORK;
+
+public class IndexUnknownIgnoreTest extends UpgradeTestBase
+{
+
+    private static final String LOG_ERROR = "Cannot find index type org.apache.cassandra.index.sasi.SASIIndex, but '"
+            + INDEX_UNKNOWN_IGNORE.getKey() + "' is true so creating noop index index_name";
+
+    @Test
+    public void testIndexUnknownIgnored() throws Throwable
+    {
+        try
+        {
+            setupClusterWithSasi()
+                .runBeforeClusterUpgrade((cluster) -> INDEX_UNKNOWN_IGNORE.setBoolean(true))
+                .runAfterClusterUpgrade((cluster) -> {
```

Comment:
We should probably add here some check to test `NoopIndex`, similar to the ones for the original `SASIIndexTest`. Probably just doing an `INSERT`, a `SELECT` query targeting the indexed column without `ALLOW FILTERING`, that will fail, and one with `ALLOW FILTERING`, that will succeed.

Reply:
those statements are correct.  it's just that the index should be on the `ck` column, not the `v` column.